wallet-new/src/Cardano/Wallet/Kernel/Transactions.hs

```diff
@@ -464,9 +464,12 @@ mkSigner spendingPassword (Just esk) snapshot addr =
                                              esk
                                              accountIndex
                                              addressIndex
+            -- eks address fix - we need to use the esk as returned
```

Comment:
Minor typo, that's `esk`. I would omit the "esk address fix" part completely, We can just say "we need to use the ESK..." exactly as you are doing now.